CONTRIBUTING.md

```diff
@@ -231,22 +227,16 @@ This library uses [semantic versioning](http://semver.org/). For each release, t
-  - The command you will need to run will look something like this: `github_changelog_generator -u ably -p ably-java --since-tag v1.2.3 --output delta.md --token $GITHUB_TOKEN_WITH_REPO_ACCESS`. Generate token [here](https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token).
-  - Using the command above, `--output delta.md` writes changes made after `--since-tag` to a new file.
-  - The contents of that new file (`delta.md`) then need to be manually inserted at the top of the `CHANGELOG.md`, changing the "Unreleased" heading and linking with the current version numbers.
-  - Also ensure that the "Full Changelog" link points to the new version tag instead of the `HEAD`.
+   - The command you will need to run will look something like this: `github_changelog_generator -u ably -p ably-java --since-tag v1.2.3 --output delta.md --token $GITHUB_TOKEN_WITH_REPO_ACCESS`. Generate token [here](https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token).
+   - Using the command above, `--output delta.md` writes changes made after `--since-tag` to a new file.
+   - The contents of that new file (`delta.md`) then need to be manually inserted at the top of the `CHANGELOG.md`, changing the "Unreleased" heading and linking with the current version numbers.
+   - Also ensure that the "Full Changelog" link points to the new version tag instead of the `HEAD`.
 4. Commit [CHANGELOG](./CHANGELOG.md)
 5. Make a PR against `main`
 6. Once the PR is approved, merge it into `main`
-7. From the updated `main` branch on your local workstation, assemble and upload:
-    1. Run `./gradlew publishToMavenCentral` to build and upload `ably-java` and `ably-android` to Nexus staging repository
-    2. Find the new staging repository using the [Nexus Repository Manager](https://oss.sonatype.org/#stagingRepositories)
-    3. Check that it contains `ably-android` and `ably-java` releases
-    4. "Release" it - this will take a few minutes during which time it will say (after a refresh of your browser) that "Activity: Operation in Progress". You can allow it to "automatically drop" after successful release. A refresh or two later of the browser and the staging repository will have disappeared from the list (i.e. it's been dropped which implies it was released successfully)
-    7. A [search for Ably packages](https://oss.sonatype.org/#nexus-search;quick~io.ably) should now list the new version for both `ably-android` and `ably-java`
-8. Add a tag and push to origin - e.g.: `git tag v1.2.4 && git push origin v1.2.4`
-9. Create the release on Github including populating the release notes
-10. Create the entry on the [Ably Changelog](https://changelog.ably.com/) (via [headwayapp](https://headwayapp.co/))
+7. Create the release and the release tag on Github including populating the release notes
```

Comment:
About step 7, I would suggest adding explicit commands 
```
7. Add a tag and push to origin - e.g.: `git tag v1.2.4 && git push origin v1.2.4` and create release with notes from the latest CHANGELOG.
```
- It's very easy for someone to mistakenly push tag as 1.2.4 instead of v1.2.4, since some SDKs follow this convention

Reply:
I believe it's easier to do right in the GitHub while writing release notes and it's less likely you forget format, since you see all previous releases

Reply:
Sure, I will approve this PR for now. 
Hopefully, there shouldn't be issues related to this in the future